src/pages/DetailPage.js

```diff
@@ -14,27 +14,31 @@ export default function DetailPage({ concert, onNavigate, pageName }) {
     <div>
       <Heading pageName={pageName} />
       <Wrapper>
-        <Image src={concert.image} alt="" />
+        <Image src={concert.image16_9} alt="" />
```

Comment:
Alternative wording:
```suggestion
        <Image src={concert.image16x9} alt="" />
```